GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -81,6 +83,8 @@ public class NuGetUpgrader : ProductUpgrader
 
         public string DownloadedPackagePath { get; private set; }
 
+        public override bool SupportsAnonymousVersionQuery { get => false; }

```

Comment:
Why do we need to support the `SupportsAnonymousVersionQuery` is `false` scenario?

Could we make it a requirement that if your organization wants to upgrade via NuGet you must support an anonymous version query endpoint?